TimePlatformer/Assets/Scripts/TimeController.cs

```diff
@@ -58,7 +58,7 @@ private TimeController()
     /// <param name="newTime"></param>
     public void ChangeTime(TimeOfDay newTime)
     {
-        Debug.Log("Time Changed!!!!" + newTime);
+        //Debug.Log("Time Changed!!!!" + newTime);
```

Comment:
Just delete the line rather than commenting it out. If needed Git is a Version Control System and we can revert changes.